api/request.go

```diff
@@ -157,6 +157,27 @@ type SliceRequest struct {
 
```

Comment:
Irrelevant about slice docs:

> See model SliceData for more info on request parameters.

> See the Metadata data model.

Seem quite rusted.


Also might need to update some other parts of the docs:
Geospatial: 

> For inline/crossline slices this is a linestring, while for time/depth slices this is essentially the bounding box of the volume.

now with "unless you supplied bounds"